src/app/components/administration/admin-settings/admin-settings-user-policy/admin-settings-user-policy.component.ts

```diff
@@ -8,15 +10,57 @@ import {userPolicyDataStore} from "../../../../shared/store/user-policy-data-sto
 })
 export class AdminSettingsUserPolicyComponent implements OnInit {
   markdownContent: string = '';
-  markdownData: any = userPolicyDataStore;
+  markdownData: any;
+  saveBtnDisabled: boolean = false
+
+  constructor(private userPolicyService: UserPolicyService, private matSnackBar: MatSnackBar) {
+  }
 
   ngOnInit(): void {
-    if (this.markdownData !== null) {
-      this.markdownContent = this.markdownData[0].markdownContent;
+    this.loadUserPolicy();
+  }
+
+  loadUserPolicy() {
+    this.userPolicyService.getAllUserPolicies().subscribe(
+      (data: any) => {
+        this.markdownData = data;
+        if (this.markdownData !== null && this.markdownData.length > 0) {
+          this.markdownContent = this.markdownData[0].markdownContent;
+        }
+        return this.markdownData
+      }
+    )
+  }
+
+  loadDate() {
+    const date = new Date().toString();
+    return date.split(' ').slice(1,4).join(' ');
+  }
+
+  savePolicy() {
+    if (this.markdownData.length > 0) {
```

Comment:
make a condition to check policy is already in database or not. this will help to avoid overwriting data in database